### PR TITLE
feat(x/twap): modify cli to add geometric option

### PR DESCRIPTION
**Pull request by p0mvn**
Closes: #3801 

## What is the purpose of the change

Adds geometric twap CLI by implementing a new command with the same args as the old arithmetic command.

Did basic test checking that command was correctly wired up to the binary. I'm tracking proper manual localosmosis tests to be done separately in the future. This will be done once edge case tests are added: https://github.com/osmosis-labs/osmosis/issues/3541)

### CODE REVIEW

**Comment by ValarDragon** [comment]
I'd rather add this as a flag than an arg

File: x/twap/client/cli/query.go [hunk 1]
```diff
@@ -85,6 +119,30 @@ Example:
 	return cmd
 }
 
+// getQuoteDenomFromLiquidity gets the quote liquidity denom from the pool. In addition, validates that base denom
+// exists in the pool. Fails if not.
+func getQuoteDenomFromLiquidity(ctx context.Context, clientCtx client.Context, poolId uint64, baseDenom string) (string, error) {
```
Comment (p0mvn) [inline x/twap/client/cli/query.go:124]: note: this was extracted from pre-existing code. No additional changes were made

**Comment by p0mvn** [comment]
> I'd rather add this as a flag than an arg

I made a separate command while trying to minimize code duplication. Felt like better UX. Please let me know what you think

**Review by ValarDragon — APPROVED** [review]
Nice

**Comment by ValarDragon** [comment]
I worry about breaking the old CLI though, can we perhaps setup a viper alias?

**Comment by p0mvn** [comment]
Alias added. Going to merge
